src/game/getAverageItemLevel.ts

```diff
@@ -0,0 +1,9 @@
+import GEAR_SLOTS from './GEAR_SLOTS';
+
+// Some Tabards have a higher ilvl than 1 so exclude them from the avg ilvl (not sure about shirts, but including them)
+const EXCLUDED_ITEM_SLOTS = [GEAR_SLOTS.SHIRT, GEAR_SLOTS.TABARD];
+
+export default function getAverageItemLevel(gear: any[]) {
```

Comment:
gear can be typed as an `Item[]` coming from here => `import { Item } from 'parser/core/Events';`